### ISSUE TITLE
Add usage instructions for .NET binding

### ISSUE THREAD

**Issue by reuben** [mozilla/DeepSpeech#2948]
Currently we only have build instructions, mentions of copying binaries around, and no mention of NuGet.

**Comment by kdavis-mozilla** [comment]
@carlfm01 Would you be willing to take this?

**Comment by carlfm01** [comment]
@kdavis-mozilla Yes, I can take it

**Comment by honkmother** [comment]
Maybe you should fix the NuGet package first, so that the instructions would work.

**Comment by carlfm01** [comment]
Yes, a reminder to me: Look into NuGet readme https://github.com/NuGet/Home/wiki/Embedding-and-displaying-NuGet-READMEs

**Comment by reuben** [comment]
@carlfm01 I think it'd be better to have the usage documentation centralized on readthedocs, as we've been moving everything to it. We used to have some stuff embedded on README's but it made it easier for changes to slip through and things to get out of sync.

**Comment by carlfm01** [comment]
Oh ok, it will be a redirect then
